src/app/views/interactives/InteractivesShow.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { getInteractive, resetSuccessMessage } from "../../actions/interactives";
+import ButtonWithShadow from "../../components/button/ButtonWithShadow";
+import BackButton from "../../components/back-button";
+import FooterAndHeaderLayout from "../../components/layout/FooterAndHeaderLayout";
+import AlertSuccess from "../../components/alert/AlertSuccess";
+
+export default function InteractivesShow(props) {
+    const dispatch = useDispatch();
+    const { successMessage, interactive } = useSelector(state => state.interactives);
+    const { rootPath } = useSelector(state => state.state);
+
+    const [successCreate, setSuccessCreate] = useState(false);
+
+    const [interactiveId, setInteractiveId] = useState("");
+    const [url, setUrl] = useState("");
+
+    useEffect(() => {
+        const { interactiveId } = props.params;
```

Comment:
`interactiveId` should be a dependency here  as  its presence is required

Reply:
done